content/en/observability_pipelines/set_up_pipelines/dual_ship_logs/http_server.md

```diff
@@ -0,0 +1,253 @@
+
+{{% observability_pipelines/processors/sensitive_data_scanner %}}
+
+{{% /tab %}}
+{{% tab "Add hostname" %}}
+
+{{% observability_pipelines/processors/add_hostname %}}
+
+{{% /tab %}}
+{{% tab "Parse JSON" %}}
+
+{{% observability_pipelines/processors/parse_json %}}
+
+{{% /tab %}}
+{{% tab "Enrichment table" %}}
+
+{{% observability_pipelines/processors/enrichment_table %}}
+
+{{% /tab %}}
+{{% tab "Generate metrics" %}}
+
+{{% observability_pipelines/processors/generate_metrics %}}
+
+{{% /tab %}}
+{{% tab "Set env vars" %}}
+
+{{% observability_pipelines/processors/set_env_vars %}}
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Install the Observability Pipelines Worker
+1. Select your platform in the **Choose your installation platform** dropdown menu.
+1. Enter the HTTP/S server address. The Observability Pipelines Worker listens to this socket address for logs from the HTTP/S server forwarder. For example, `0.0.0.0:9997`.
+1. Provide the environment variables for each of your selected destinations. See [prerequisites](#prerequisites) for more information.
+{{< tabs >}}
+{{% tab "Datadog" %}}
+
+{{% observability_pipelines/destination_env_vars/datadog %}}
+
+{{% /tab %}}
+{{% tab "Splunk HEC" %}}
+
+{{% observability_pipelines/destination_env_vars/splunk_hec %}}
+
+{{% /tab %}}
+{{% tab "Sumo Logic" %}}
+
+{{% observability_pipelines/destination_env_vars/sumo_logic %}}
+
+{{% /tab %}}
+{{% tab "Syslog" %}}
+
+{{% observability_pipelines/destination_env_vars/syslog %}}
+
+{{% /tab %}}
+{{% tab "Chronicle" %}}
+
+{{% observability_pipelines/destination_env_vars/chronicle %}}
+
+{{% /tab %}}
+{{% tab "Elasticsearch" %}}
+
+{{% observability_pipelines/destination_env_vars/elasticsearch %}}
+
+{{% /tab %}}
+{{% tab "OpenSearch" %}}
+
+{{% observability_pipelines/destination_env_vars/opensearch %}}
+
+{{% /tab %}}
+{{% tab "Amazon OpenSearch" %}}
+
+{{% observability_pipelines/destination_env_vars/amazon_opensearch %}}
+
+{{% /tab %}}
+{{% tab "New Relic" %}}
+
+{{% observability_pipelines/destination_env_vars/new_relic %}}
+
+{{% /tab %}}
+{{< /tabs >}}
+1. Follow the instructions for your environment to install the Worker.
+{{< tabs >}}
+{{% tab "Docker" %}}
+
+{{% observability_pipelines/install_worker/docker %}}
+
+{{% /tab %}}
+{{% tab "Amazon EKS" %}}
+
+{{% observability_pipelines/install_worker/amazon_eks %}}
+
+{{% /tab %}}
+{{% tab "Azure AKS" %}}
+
+{{% observability_pipelines/install_worker/azure_aks %}}
+
+{{% /tab %}}
+{{% tab "Google GKE" %}}
+
+{{% observability_pipelines/install_worker/google_gke %}}
+
+{{% /tab %}}
+{{% tab "Linux (APT)" %}}
+
+{{% observability_pipelines/install_worker/linux_apt %}}
+
+{{% /tab %}}
+{{% tab "Linux (RPM)" %}}
+
+{{% observability_pipelines/install_worker/linux_rpm %}}
+
+{{% /tab %}}
+{{% tab "CloudFormation" %}}
+
+{{% observability_pipelines/install_worker/cloudformation %}}
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Send logs to the Observability Pipelines Worker over Http_server
```

Comment:
`over Http_server` was removed for all six docs in this PR: https://github.com/DataDog/documentation/pull/25475/commits/e71f1d162554525aa12a7bbf263717f0b75edf49